angular-ngrx-scss/src/app/state/repository/repository.selectors.ts

```diff
@@ -1,26 +1,26 @@
 import { createFeatureSelector, createSelector } from '@ngrx/store';
-import { RepoState } from './repository.state';
+import { RepositoryState } from './repository.state';
 
-export const repositoryFeatureKey = 'repo';
+export const repositoryFeatureKey = 'repository';
 export const selectRepositoryState =
-  createFeatureSelector<RepoState>(repositoryFeatureKey);
+  createFeatureSelector<RepositoryState>(repositoryFeatureKey);
 
 export const selectedRepository = createSelector(
   selectRepositoryState,
```

Comment:
why do we even have this selector if it's not actually selecting anything different than the whole state?

Reply:
Valid point. It's used in two different component files, but I agree that it doesn't really seem super necessary. I'll make a note to check into this whenever this section of the store is refactored.

Reply:
can we put a TODO comment in here to not forget about this?

Reply:
Added!